internal/version/main.go

```diff
@@ -0,0 +1,51 @@
+package version
+
+import (
+	"fmt"
+	"runtime"
+)
+
+var (
+	version   string
+	buildDate string
+	tempo     string
+)
+
+// Version holds this Operator's version as well as the version of some of the components it uses.
+type Version struct {
+	Operator  string `json:"tempo-operator"`
```

Comment:
As Tempo and Go are suffixed with `-version`, I'd also suffix the operator field with `-version`.